packages/cli/generators/openapi/schema-helper.js

```diff
@@ -334,6 +317,58 @@ function generateModelSpecs(apiSpec, options) {
   return models;
 }
 
+/**
+ * Register the named schema
+ * @param {string} schemaName Schema name
+ * @param {object} schema Schema object
+ * @param {object} typeRegistry Options for objectTypeMapping & schemaMapping
+ */
+function registerSchema(schemaName, schema, typeRegistry) {
+  if (typeRegistry.objectTypeMapping.get(schema)) return;
+  if (!schemaName) {
+    // Anonymous schema
+    if (!typeRegistry.promoteAnonymousSchemas) {
+      // Skip anonymous schemas
+      return;
+    }
+    // Infer a schema name
+    if (schema.title) {
+      schemaName = camelCase(schema.title);
+    } else {
+      if (typeRegistry.nameIndex == null) typeRegistry.nameIndex = 0;
+      typeRegistry.nameIndex++;
+      schemaName = 'AnonymousType_' + typeRegistry.nameIndex;
```

Comment:
Can we please infer a more descriptive schema name than "Anonymous_XYZ", perhaps using operation name and argument name.

For example, in the USPTO spec, the operation `list-searchable-fields` uses several anonymous schemas, they can be called:
 - the first parameter: `ListSearchableFields_Dataset`
 - the second parameter: `ListSearchableFields_Version`
 - the third parameter: `ListSearchableFieldsResponse` (not using `_` to avoid clash with a parameter called "Response")